site2/docs/functions-package-python.md

```diff
@@ -16,16 +16,16 @@ To package a Python function into **one Python file**, complete the following st
 1. Write a Python function.
 
    ```python
-    from pulsar import Function #  import the Function module from Pulsar
-
-    # The classic ExclamationFunction that appends an exclamation at the end
-    # of the input
-    class ExclamationFunction(Function):
+   from pulsar import Function #  import the Function module from Pulsar
+   
+   # The classic ExclamationFunction that appends an exclamation at the end
+   # of the input
+   class ExclamationFunction(Function):
       def __init__(self):
-        pass
-
+         pass
+      
```

Comment:
extra spaces ?

Reply:
Yes, python default indent is 4 spaces, so I add extra spaces. I've tested it, users can copy & paste this code snippet to a file directly.

Reply:
No, I mean there are spaces in L26 that I believe should not be there since the line is empty

Reply:
Understand. I have removed them.